### DISCUSSION TITLE
ページネーションのページ数がおかしい

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#471] [要望・アイデア・不具合]
https://qiita.com/search?q=user%3Ajnchito%20rails%20order&sort=&stocked=&page=3 を開いて4ページ目に進もうとすると

> 「user:jnchito rails order」に一致する記事は見つかりませんでした。

が表示されました。

![kYSkWOipRO](https://user-images.githubusercontent.com/1148320/235384234-e65d55ee-38aa-4b62-8665-d2177feb1ab4.gif)

**Comment by tsubolabo** [comment]


**Reply by tanaka-asami415** [reply]
@JunichiIto @tsubolabo
ご不便をおかけして申し訳ありません。
修正する予定ではあるのですが、直近の開発優先度などから考えると先になってしまいそうです🙏

**Reply by kyntk** [reply]
@JunichiIto お待たせいたしました。
ページネーション数の不具合を解消しました！
https://qiita.com/search?q=rails%20created%3A%3E2023-04-09%20stocks%3A%3E1&sort=&stocked=&page=3

**Reply by JunichiIto** [reply]
@kyntk ご対応ありがとうございます！修正されているのを確認しました😄
